testdata/storage/fibrechannel/pvc1.yaml

```diff
@@ -0,0 +1,17 @@
+{
+    "apiVersion": "v1",
+    "kind": "PersistentVolumeClaim",
```

Comment:
Do we need a pvc for each kind of storage ? Can't we reuse exiting one ?

Reply:
Move these test files to cucushift. Close this pr. Thanks.